docs/contributing/coding-guidelines/ros-nodes/directory-structure.md

```diff
@@ -3,3 +3,52 @@
 !!! warning
 
     Under Construction
+
+## C++ package
+
+```txt
+<package_name>
+├─ config
+│   └─ foo.param.yaml
```

Comment:
Considering https://github.com/autowarefoundation/autoware-documentation/pull/224/files#r1005413621, should we write like this? :thinking: 

```suggestion
│   ├─ foo_ros.param.yaml
│   └─ foo_non_ros.yaml
```